apps/docs/src/guide/fuels-cli/commands.md

```diff
@@ -61,7 +61,7 @@ npx fuels@{{fuels}} help build
 
 ```console
 Options:
-  -p, --path <path>  Path to project root (default: "/Users/anderson/Code/fuel/fuels-ts/apps/docs")
+  --path <path>  Path to project root (default: "/Users/anderson/Code/fuel/fuels-ts/apps/docs")
```

Comment:
How come this was changed? All the other flags still follow the shorthand/longhand convention? I don't think we should change this.